src/encoded/static/components/gene.js

```diff
@@ -77,49 +77,13 @@ const Gene = (props) => {
                         </dd>
                     </div>
 
-                    {context.go_bp_ids ?
-                      <div data-test="go_ids">
-                          <dt>Biological Process Ontology</dt>
-                          <dd>
-                              <ul>
-                                  {context.go_bp_ids.map(goBPID =>
-                                      <li key={goBPID}>
-                                          <span>{goBPID}</span>
-                                      </li>
-                                  )}
-                              </ul>
-                          </dd>
-                      </div>
-                    : null}
-
-                    {context.go_cc_ids ?
-                      <div data-test="go_ids">
-                          <dt>Cellular Component Ontology</dt>
-                          <dd>
-                              <ul>
-                                  {context.go_cc_ids.map(goCCID =>
-                                      <li key={goCCID}>
-                                          <span>{goCCID}</span>
-                                      </li>
-                                  )}
-                              </ul>
-                          </dd>
-                      </div>
-                    : null}
-
-                    {context.go_mf_ids ?
-                      <div data-test="go_ids">
-                          <dt>Molecular Function Ontology</dt>
-                          <dd>
-                              <ul>
-                                  {context.go_mf_ids.map(goMFID =>
-                                      <li key={goMFID}>
-                                          <span>{goMFID}</span>
-                                      </li>
-                                  )}
-                              </ul>
-                          </dd>
-                      </div>
+                    {context.go_annotations ?
```

Comment:
_Might_ want to make it `context.go_annotations && context.go_annotations.length > 0` here, but it depends if you can have an empty array here or not. If you can’t, then no reason to change this.

Reply:
Thanks.